prime-router/src/test/kotlin/unittest/UnitTestUtils.kt

```diff
@@ -0,0 +1,30 @@
+package gov.cdc.prime.router.unittest
+
+import gov.cdc.prime.router.Element
+import gov.cdc.prime.router.Metadata
+import gov.cdc.prime.router.Schema
+
+/**
+ * Utilities specific to unit testing.
+ */
+class UnitTestUtils {
+    companion object {
+        /**
+         * A simple schema for testing.
+         */
+        val simpleSchema = Schema(name = "one", topic = "test", elements = listOf(Element("a"), Element("b")))
+
+        /**
+         * A simple metadata instance that does not use the real configuration.
+         */
+        val simpleMetadata by lazy {
+            Metadata(simpleSchema)
+        }
+
+        @Deprecated(
```

Comment:
Note the deprecated here.  We need to move or redo any tests that require a full metadata catalog.  Once we can do that we need to remove this variable and the other metadata files this uses.